wrench/reftests/text/reftest.list

```diff
@@ -21,7 +21,7 @@ fuzzy(1,100) == decorations-suite.yaml decorations-suite.png
 fuzzy(1,735) == shadow-grey.yaml shadow-grey-ref.yaml
 fuzzy(1,614) == shadow-grey-transparent.yaml shadow-grey-ref.yaml
 == subtle-shadow.yaml subtle-shadow-ref.yaml
-== shadow-atomic.yaml shadow-atomic-ref.yaml
+options(disable-aa) == shadow-atomic.yaml shadow-atomic-ref.yaml
```

Comment:
Why did this need to change? Could it mean that we're accidentally drawing subpixel in the shadow or something like that?

Reply:
That is one part of it, yes. Then the further part is that due to differing preblending applied depending on text color, it won't necessarily match either even with grayscale. The only mode that can actually match on all backends for a fast shadow of a given color and normal text of that color, is mono, essentially.

Reply:
@Gankro do you see any issues with this change?